pkg/resource/deploy/deploytest/languageruntime.go

```diff
@@ -77,7 +77,25 @@ func (p *languageRuntime) Run(info plugin.RunInfo) (string, bool, error) {
 	// Run the program.
 	done := make(chan error)
 	go func() {
-		done <- p.program(info, monitor)
+		var err error
```

Comment:
Should we initalize this to something like "program panicked" so its even more clear what happened?